File: CTDK_Python/146/_146.py
n = int(input("Nhap n: "))

dn = 0
t = n
while t != 0:
    dv = t % 10
    dn = dn * 10 + dv
    t = t // 10

if dn == n:
    print("So doi xung")
else:
    print("Khong la so doi xung")

File: LTHAM/146/146.cpp
#include <iostream>
using namespace std;

int ktDoiXung(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	if (ktDoiXung(n) == 1)
		cout << "Doi xung" << endl;
	else
		cout << "Khong phai doi xung" << endl;
	return 0;
}

int ktDoiXung(int n)
{
	int dn = 0;
	int t = n;
	while (t != 0)
	{
		int dv = t % 10;
		dn = dn * 10 + dv;
		t = t / 10;
	}
	
	if (dn == n)
		return 1;
	return 0;
}
